src/providers/authProvider/index.ts

```diff
@@ -2,25 +2,23 @@ import { type AuthProvider, type DataProvider } from 'react-admin'
 import constants from '../../constants'
 import { AuditType, trackEvent } from '../../utils/audit'
 import { decryptData, encryptData, generateSalt } from '../../utils/ecnryption'
-export const getToken = (): string | null => {
-  const JsonUser = localStorage.getItem(constants.TOKEN_KEY)
-  if (JsonUser !== null) {
-    const user = JSON.parse(JsonUser) as User & { salt: string }
-    const decryptedData = decryptData(`${user.id}${user.salt}`)
-    user.id = decryptedData.substring(
-      0,
-      decryptedData.length - user.salt.length
-    )
-    return JSON.stringify(user)
-  } else return null
+export const getUser = (): string | undefined => {
```

Comment:
could this return an instance of `User` please?  Then the rest of the app doesn't need to know how we store the user data.

Reply:
It was already there (returning string) but I can return user.

Reply:
`getUser = (): User | undefined`